website/content/en/v0.32/contributing/development-guide.md

```diff
@@ -151,12 +151,6 @@ export KO_DOCKER_REPO="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws
 aws ecr get-login-password --region "${AWS_DEFAULT_REGION}" | docker login --username AWS --password-stdin "${KO_DOCKER_REPO}"
 ```
 
-Finally, to deploy the correct IAM permissions, including the instance profile for provisioned nodes, run
-
-```bash
-make setup
-```
-
```

Comment:
I think v0.32.0 still has this command in the Makefile

Reply:
Yes it does - wasn't sure about keeping or removing this one tbh, since this is part of the development setup, not the actual software. Happy to add it again tho